### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by mrocklin**
cc @martindurant

### CODE REVIEW

**Comment by mrocklin** [comment]
I've removed the WIP label.  @martindurant ,  review appreciated

File: hdfs3/tests/test_hdfs3.py [hunk 1]
```diff
@@ -456,3 +503,133 @@ def test_different_handles_in_processes():
...
+    hdfs.chmod(a, 'w')
+
+    with hdfs.open(a, 'a') as f:
+        pass
+
+
+@pytest.mark.xfail
+def test_chown(hdfs):
+    hdfs.touch(a)
+    i = hdfs.info(a)
```
Comment (mrocklin) [inline hdfs3/tests/test_hdfs3.py]: I'm not sure what's going on in the xfailed tests.  @martindurant were you ever able to get this functionality to work?

**Comment by mrocklin** [comment]
Merging.  Review still appreciated.

**Comment by martindurant** [comment]
I’ll look into it, but probably not today. As far as I remember, chmod works fine, but chown of course will require a suitable user/group to refer to.
I’ll update the tests.

**Pull request merged**